integrations/destinations/postgresql.mdx

```diff
@@ -170,6 +181,30 @@ GROUP BY
 ```
 
 ### Sink from RisingWave
+
+You can use either the JDBC connector or the native PostgreSQL connector to sink data to PostgreSQL.
+
+#### Option 1: Native PostgreSQL Connector (Recommended for PostgreSQL-specific features)
+
+Use the native PostgreSQL connector for better performance and PostgreSQL-specific features:
```

Comment:
```suggestion
Use the native PostgreSQL connector for PostgreSQL-specific features:
```

Performance for JDBC and PG native sink is actually comparable. I think we need to rewrite this in a way that highlights we are sunsetting JDBC sink (in a backwards-compatible way). You can remove the changes to this file first, and assign another documentation issue to me to update it.

Reply:
> I think we need to rewrite this in a way that highlights we are sunsetting JDBC sink (in a backwards-compatible way).

Oh really? Then we need dedicated connectors for each jdbc compatible downstream?